libs/common-components/src/lib/loading-mask-interceptor.service.ts

```diff
@@ -0,0 +1,19 @@
+import {Injectable} from '@angular/core';
+import {HttpEvent, HttpHandler, HttpInterceptor, HttpRequest} from "@angular/common/http";
+import {Observable} from "rxjs";
+import {finalize} from 'rxjs/operators'
+import {LoadingMaskService} from "./loading-mask.service";
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LoadingMaskInterceptorService implements HttpInterceptor {
+
+  constructor(private _loadingMaskService: LoadingMaskService) {
+  }
+
+  intercept(req: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
```

Comment:
some requests dont need loading, how do you handle this?

Reply:
In the current solution i don't handle such case, probably @egorovpavel could clarify a bit - should we add this feature in this PR.

Reply:
> some requests don't need loading

Every XHR request needs loading indication. Maybe not all need fullscreen indication, but all XHR requests must have some sort of indication.

Reply:
@egorovpavel we have also 'shadow' requests, such as getFonts, getFormats, so I dont think that all requests must have the indication